nodes/core/core/20-inject.js

```diff
@@ -26,27 +26,37 @@ module.exports = function(RED) {
         this.repeat = n.repeat;
         this.crontab = n.crontab;
         this.once = n.once;
+        this.onceDelay = n.onceDelay || 0.1; // seconds
         var node = this;
         this.interval_id = null;
         this.cronjob = null;
 
-        if (this.repeat && !isNaN(this.repeat) && this.repeat > 0) {
-            this.repeat = this.repeat * 1000;
-            if (RED.settings.verbose) { this.log(RED._("inject.repeat",this)); }
-            this.interval_id = setInterval( function() {
-                node.emit("input",{});
-            }, this.repeat );
-        } else if (this.crontab) {
-            if (RED.settings.verbose) { this.log(RED._("inject.crontab",this)); }
-            this.cronjob = new cron.CronJob(this.crontab,
-                function() {
-                    node.emit("input",{});
-                },
-                null,true);
-        }
+        node.factorSecondsToMilliseconds = 1000;
```

Comment:
Why bother ? Adding another property to every inject node ?  just leave it as a constant 1000...

Reply:
Dave, can I use const or should I use var here?

Reply:
don't use anything... just use  * 1000 where needed.

Reply:
even line 29 would be easier as this.onceDelay = (n.onceDelay || 0.1) * 1000;